src/main.css

```diff
@@ -322,6 +322,10 @@ button[disabled] {
 }
 .cell {
     display: table-cell;
+    vertical-align: middle;
```

Comment:
@negar-binary This is a sensitive change. In video, I am doing inline editing for chartOptions cell ONLY